server/rpm/pbench-server.spec.j2

```diff
@@ -84,6 +84,17 @@ cd /%{installdir}
 rm -rf node_modules
 npm install
 
+# this only handles v0.3
+# AFAIK, we don't need all the other modules that package.json includes.
+jslist="node_modules/d3/d3.min.js
+        node_modules/d3-queue/build/d3-queue.min.js
+        node_modules/save-svg-as-png/lib/saveSvgAsPng.js"
+
+# copy them to the appropriate directories for v0.3
+for x in ${jslist} ;do
+    cp $x /%{installdir}/%{static}/js/v0.3
```

Comment:
Does there need to be an explicit create of the `/js/v0.3` directory?

Reply:
Looking at it more closely: it does not need an explicit create because the `cp -a` of the web-server stuff creates that subdirectory, but it is hidden, so it needs at least a comment (and maybe an explicit mkdir).

Reply:
Added both a comment and an explicit `mkdir -p` which *should* be a no-op.

Reply:
Thanks; the clarification is good.